amti/clis/preview.py

```diff
@@ -0,0 +1,100 @@
+"""CLI for running a simple web server to preview HIT"""
+
+import os
+import json
+from http.server import BaseHTTPRequestHandler, HTTPServer
+from urllib.parse import urlparse
+
+import click
+import jinja2
+
+from amti import actions
+from amti import settings
+from amti import utils
+
+
+class MyServer(HTTPServer):
+    def __init__(
+        self,
+        server_address,
+        RequestHandlerClass,
+        template_path=None,
+        data_path=None,
+    ):
+        super().__init__(server_address, RequestHandlerClass)
+        self.template_path = template_path
+        self.data_path = data_path
+
+
+class MyHandler(BaseHTTPRequestHandler):
+    def _set_headers(self):
+        self.send_response(200)
+        self.send_header('Content-type', 'text/html')
+        self.end_headers()
+
+    def do_GET(self): # handle GET request
+        self._set_headers()
+
+        # Get data index from query (optional)
+        query = urlparse(self.path).query
+        try:
+            query_components = dict(qc.split("=") for qc in query.split("&"))
+        except ValueError:
+            query_components = {}
+        index = int(query_components.get('id', '0'))
```

Comment:
Using the URL to enable viewing different HITs is a great idea 😄! I've thought about building a preview server, but I'll admit I've been overlooking the fact that people might want to view different HITs.